core/src/test/resources/HistoryServerExpectations/executor_list_json_expectation.json

```diff
@@ -14,6 +14,7 @@
   "totalDuration" : 8820,
   "totalGCTime" : 352,
   "totalInputBytes" : 28000288,
+  "totalOutputBytes" : 0,
```

Comment:
It would be nice to see at least one example where the totalOutputByte is nonzero. For example you could modify an existing spark events in `core/src/test/resources/spark-events` as a new test test could be overkill for this.